packages/notebook-extension/src/celltaglist.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { NotebookPanel } from '@jupyterlab/notebook';
+import React from 'react';
+//import PropTypes from 'prop-types';
+import { ReactWidget, VDomModel, VDomRenderer } from '@jupyterlab/apputils';
+import { CommandRegistry } from '@lumino/commands';
+import { Button } from '@jupyterlab/ui-components';
+import { ITranslator } from '@jupyterlab/translation';
+import { IRenderMime } from '@jupyterlab/rendermime';
+
+export class FilterButtonWidget extends ReactWidget {
+  private _commands: CommandRegistry;
+  private _commandID: string;
+  private _trans: IRenderMime.TranslationBundle;
+
+  constructor(
+    commands: CommandRegistry,
+    commandID: string,
+    translator: ITranslator
+  ) {
+    super();
+    this._commands = commands;
+    this._commandID = commandID;
+    this._trans = translator.load('jupyterlab');
+  }
+  onClick = () => this._commands.execute(this._commandID);
+  render() {
+    return (
+      <>
+        <p>{this._trans.__(this._commands.label(this._commandID))}</p>
+        <Button onClick={this.onClick}>{'Open the filtering tool'}</Button>
```

Comment:
It should be the other way round. Translation should be applied in the second line as it has a string literal. Translation has no effect on strings in variables/returned by functions. This is because extraction is based on static analysis  please see https://jupyterlab.readthedocs.io/en/stable/extension/internationalization.html

```suggestion
        <p>{this._commands.label(this._commandID)}</p>
        <Button onClick={this.onClick}>{this._trans.__('Open the filtering tool')}</Button>
```